library/src/test/java/com/datatorrent/lib/db/jdbc/JdbcOperatorTest.java

```diff
@@ -375,14 +411,37 @@ public void testJdbcPojoInputOperator()
     OperatorContextTestHelper.TestIdOperatorContext context = new OperatorContextTestHelper.TestIdOperatorContext(
         OPERATOR_ID, attributeMap);
 
-    insertEventsInTable(10);
```

Comment:
I would still make it a function as the test case becomes more readable with this function.